openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -723,6 +723,39 @@ response_type=code
 &interaction_types_supported=openid4vp_presentation,redirect_to_web
 ```
 
+The following non-normative example shows an initial request to the Interactive Authorization Endpoint with a signed request object:
+
+```http
+POST /iar HTTP/1.1
+Host: server.example.com
+OAuth-Client-Attestation: eyJ...
+OAuth-Client-Attestation-PoP: eyJ...
+Content-Type: application/x-www-form-urlencoded
+
+request=eyJrd...
+```
+
+The following non-normative example shows a payload of a signed request object:
+
+```json
+{
+  "iss": "CLIENT1234",
+  "aud": "https://server.example.com",
+  "response_type": "code",
+  "client_id": "CLIENT1234",
+  "code_challenge": "E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM",
+  "code_challenge_method": "S256",
+  "redirect_uri": "https%3A%2F%2Fclient.example.org%2Fcb",
+  "authorization_details": [
+    {
+      "type": "openid_credential",
+      "credential_configuration_id": "UniversityDegreeCredential"
+    }
+  ],
+  "interaction_types_supported": "openid4vp_presentation,redirect_to_web"
+}
```

Comment:
```suggestion
```json
{
  "iss": "CLIENT1234",
  "aud": "https://server.example.com",
  "response_type": "code",
  "client_id": "CLIENT1234",
  "code_challenge": "E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM",
  "code_challenge_method": "S256",
  "redirect_uri": "https%3A%2F%2Fclient.example.org%2Fcb",
  "authorization_details": [
    {
      "type": "openid_credential",
      "credential_configuration_id": "UniversityDegreeCredential"
    }
  ],
  "interaction_types_supported": "openid4vp_presentation,redirect_to_web",
  "nonce": "af0ifjsldkj"
}
```
what about nonce?
why it needs PKCE?

Reply:
yeah but then we should PKCE related guidance in IAR section. adding PKCE without anything in section 6 feels abrupt